nrfu_tests/test_definition.yaml

```
@@ -0,0 +1,13 @@
+- name: nrfu_tests
+  testcases:
+  - name: test_security_rp_login_banner
+    description: Testcase for verification of login banner is configured.
```

```suggestion
    description: Testcase for verification of security rp login banner
```
pls update on all applicable cases.